docs/xks/operator-guide/architecture-and-design.md

```diff
@@ -0,0 +1,117 @@
+- SSO from the tenant namespaces to the tenant resource groups to make sure no secrets have to be shared
+- Ability to use mTLS together with a service mesh (linkerd)
+- Automated certificate and dns management from the platform
+
+#### Platform
+
+- Observability of the platform handled by the platform team
+- Runtime security in the platform using Falco
+- Automated update management of Infrastructure as Code (with code review)
+
+### Access
+
+The primary authentication method to access any resource is based on Azure AD. This isn't enforced using the framework in all places, with two examples being AWS Console / CLI and GitHub. Most of the actions taken by a tenant engineer will require authentication to Azure AD either using the Azure Management Console or the Azure CLI.
+
+A tenant will be granted access to the clusters using a [proxy](https://github.com/XenitAB/azad-kube-proxy) built by Xenit and provided in the framework, making sure they don't have to reconfigure their computers when a blue/green deployment of the clusters are made and the Kubernetes API endpoint change. The proxy will move with the clusters and it will be seamless for the tenant engineers.
+
+The proxy also provides a CLI (kubectl plugin through [krew](https://krew.sigs.k8s.io/)) that makes it easier to both discover and configure access to the clusters. A valid session with Azure CLI is required to use it.
+
+Other than that, most of the access and work with the tenant resources are done through a GitOps repository for changes in regards to applications in Kubernetes and a terraform repository in regards to resources in the cloud provider.
+
+## Network design
+
+By default, the network setup is expected to be quite autonomous and usually considered to be an external service compared to everything else in the organization using it. It is possible to setup peering with internal networks, but usually it begins with a much simpler setup and then grows organically when required.
+
+<img alt="XKS Simple Network Design" src={useBaseUrl("img/assets/xks/operator-guide/simple-network-design.jpg")} />
+
+The cluster environments are completely separated from each other, but a hub in the production subscription has a peering with them to provide static IP-addresses for CI/CD like terraform to access resources.
+
+Inside an environment the cluster is using kubenet and Calico to keep the amount of IP-addresses to a minimum. Services can either be exposed internally or externally using either a service or an ingress, where most tenants exclusively use ingress (provided by ingress-nginx).
+
+Inside the clusters Calico is used to limit traffic between namespaces.
+
+## Backup
+
+The platform is built to be ephemeral wich means any cluster can at any time be wiped and a new setup without any loss of data. This means that tenants are not allowed to store state inside of the clusters and are required to store it in the cloud provider (blob storage, databases, message queues etc.).
```

Comment:
I think here (and possible elsewhere) XKF would be better.

Reply:
done!